do.sh

```diff
@@ -1,286 +1,295 @@
-
-" Fixes slow scrolling on mac "
-set lazyredraw
-
-" Make y(y) and paste(p) operations use the system clipboard
-set clipboard=unnamed,unnamedplus
-
-" <leader>dt will open definition of function under cursor in new tab (vim-go) "
-au FileType go nmap <leader>dt <Plug>(go-def-tab)
-
-" Customize the statusline "
-set laststatus=2
-highlight StatuslineFilename ctermfg=Black ctermbg=DarkGreen
-highlight StatuslineModified ctermfg=DarkMagenta ctermbg=LightGreen
-highlight StatuslineNumbers ctermfg=Black ctermbg=DarkYellow
-set statusline=%#StatuslineFilename#   " Set color for file path
-set statusline+=%F                     " Full file path, at most 40 characters
-set statusline+=\                      " A space
-set statusline+=%#StatuslineModified#  " Set color for modified flag
-set statusline+=%m                     " Modified flag
-set statusline+=%#StatuslineFilename#  " Set color for the rest of the bar
-set statusline+=%=                     " Split the left and right sides
-set statusline+=%#StatuslineNumbers#   " Set color for line numbers
-set statusline+=%l,                    " Line number
-set statusline+=\                      " A space
-set statusline+=%-3c                    " Column number
-set statusline+=\ \|\                  " A separator
-set statusline+=%L                     " Total number of lines
-
-" Remove ugly Nerdtree statusline "
-let g:NERDTreeStatusline = '%#NonText#'
-
-" Rename tabs to show tab number.
-" (Based on http://stackoverflow.com/questions/5927952/whats-implementation-of-vims-default-tabline-function)
-if exists("+showtabline")
-    function! MyTabLine()
-        let s = ''
-        let wn = ''
-        let t = tabpagenr()
-        let i = 1
-        while i <= tabpagenr('$')
-            let buflist = tabpagebuflist(i)
-            let winnr = tabpagewinnr(i)
-            let s .= '%' . i . 'T'
-            let s .= (i == t ? '%1*' : '%2*')
-            let s .= ' '
-            let wn = tabpagewinnr(i,'$')
-
-            let s .= '%#TabNum#'
-            let s .= i
-            " let s .= '%*'
-            let s .= (i == t ? '%#TabLineSel#' : '%#TabLine#')
-            let bufnr = buflist[winnr - 1]
-            let file = bufname(bufnr)
-            let buftype = getbufvar(bufnr, 'buftype')
-            if buftype == 'nofile'
-                if file =~ '\/.'
-                    let file = substitute(file, '.*\/\ze.', '', '')
-                endif
-            else
-                let file = fnamemodify(file, ':p:t')
-            endif
-            if file == ''
-                let file = '[No Name]'
-            endif
-            let s .= ' ' . file . ' '
-            let i = i + 1
-        endwhile
-        let s .= '%T%#TabLineFill#%='
-        let s .= (tabpagenr('$') > 1 ? '%999XX' : 'X')
-        return s
-    endfunction
-    set stal=2
-    set tabline=%!MyTabLine()
-    set showtabline=1
-    highlight link TabNum Special
-endif
-EOM
+#!/bin/bash
+
+echo "begin install vim"

```

Comment:
This comment is misleading if you are not in Windows (Because we haven't started installing vim and we will ask you where you want to install it first)

If you want, you can add this message in the Windows section.

```suggestion
echo "Installing Vim"
```